src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -23,6 +23,9 @@
             + "Example: " + COMMAND_WORD + " n/alice n/bob t/friends t/colleagues";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": " + COMMAND_DESCRIPTION + COMMAND_EXAMPLE;
+    public static final String CASE_SENSITIVE_FLAG_FORMAT_MESSAGE = "The case-sensitive flag `c/` must come "
+            + "right after the " + "command word!\n" + "For example, rather than 'find n/NAME c/ t/TAG' or "
+            + "'find n/NAME t/TAG c/', it should be 'find c/ n/NAME t/TAG' instead.";
```

Comment:
I'm not sure if this issue has been raised by if the user is forced to put c right after find wouldnt it be better to change the command word to findc?

Reply:
I think it'll be better to leave it as an optional flag rather than an entire command catered to it. findc and findAnyc would be too similar to find and findAny. Personally, I don't think there is enough justification to create these commands. What about the rest?

Reply:
I think its too late to change anyway HAHA